editor/import/resource_importer_scene.cpp

```diff
@@ -91,6 +91,16 @@ Ref<Animation> EditorSceneFormatImporter::import_animation(const String &p_path,
 	ERR_FAIL_V(nullptr);
 }
 
+void EditorSceneFormatImporter::get_import_options(const String &p_path, List<ResourceImporter::ImportOption> *r_options) {
+	GDVIRTUAL_CALL(_get_import_options, p_path);
+}
```

Comment:
It's not possible to return data from this function as bound.

Reply:
Yes, that API is broken.
The recommendation I have been giving for now is to create a EditorScenePostImportPlugin which adds the desired options, which can be made only visible on the supported format.

One of the tasks in 4.4 I want to do is to rework the scene and plugin apis since there are many problems: we even had a last minute problem in 4.3 because of import defaults getting broken when we changed the gltf options on the fbx importer, and that happened in part because the APIs are inconsistent and extremely confusing.